contracts/stellar-interchain-token/src/contract.rs

```diff
@@ -41,21 +41,6 @@ impl InterchainToken {
     }
 }
 
-#[contractimpl]
-impl OwnableInterface for InterchainToken {
-    fn owner(env: &Env) -> Address {
-        interfaces::owner(env)
-    }
-
-    fn transfer_ownership(env: &Env, new_owner: Address) {
-        let old_owner = Self::owner(env);
-
-        interfaces::transfer_ownership::<Self>(env, new_owner.clone());
-
-        TokenEvents::new(env).set_admin(old_owner, new_owner);
```

Comment:
Actually, this was being implemented manually since it emits an extra event that the default implementation doesn't do (to stay compatible with the token standard)

Add a notice explaining this so we don't remove it accidentally, and the event unit test should test this event to catch changes